src/components/OpenLayersMap.js

```diff
@@ -48,15 +50,19 @@ class OpenLayersMap extends React.Component {
     if (selectedTrail !== prevProps.selectedTrail && selectedTrail) {
 
       try {
-        const firstTrailGeom = trails.getIn([selectedTrail, 'features'])[0].getGeometry()
-        const geomExtent = firstTrailGeom.getExtent();
+
+        const GeomCollection = new GeometryCollection()
```

Comment:
Only classes themselves are capitalized, instances use the normal camelCase, eg. `geomCollection`